.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toxenv: [ py39, py38, py37, py36, py35, py33, py27, py26 ]
+    name: ${{ matrix.toxenv }}
+    env:
+      LD_PRELOAD: /lib/x86_64-linux-gnu/libSegFault.so
+      SEGFAULT_SIGNALS: all
+    steps:
+      - name: Checkout
+        uses: actions/checkout@main
+      - name: Setup python
+        uses: actions/setup-python@main
+        with:
+          cache: 'pip'
+      - name: Setup
+        run: |
+          if [[ ${{ matrix.toxenv }} == py2* ]]; then sudo apt-get install -y python-dev; fi
+          pip install -r requirements.txt
```

Comment:
I would keep `requirements.txt` reserved for non-test dependencies. So we can perhaps just rename it to `requirements-testing.txt` or something, or we remove it and just replace this line with `pip install tox` since it's just one dependency.

Reply:
The file is used by `setup-python`, but it's configurable. I have renamed it to `requirements-dev.txt`, so it's not limited to testing.